src/components/card/card.module.css

```
@@ -0,0 +1,43 @@
+.cardDefault{
```

```suggestion
.card {
```

Lembre do espaçamento.